src/docs/index.ts

```diff
@@ -0,0 +1,46 @@
+/**
+ * Known pages in the app. Highest order context.
+ */
+export enum DocsPageName {
+  HOME_PAGE = 'home-page',
+}
```

Comment:
Do we expect any other pages given that it's a parameter in all other places? I'm not sure if a single enum is worth it if we don't have any others we expect